src/wix/WixToolset.Core/IOptimizer.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information.
```

Comment:
You need to rebase your change so that this change is not mixed in the middle.

Reply:
Done.